v2/gax.go

```diff
@@ -35,4 +35,4 @@
 // to simplify code generation and to provide more convenient and idiomatic API surfaces.
 package gax
 
-const Version = "2.0.0"
+const Version = "2.0.1"
```

Comment:
It's too bad this is exported. Does anyone use it?

Reply:
Yes - client libraries report the gax version they're using.

Reply:
@jba I'm not sure if you have a specific concern. If it's fear that this might go outdated, we could make some kind of test to fail CI if this go out of date, etc?

Reply:
Ditto - can I resolve this? Or is there a concern?

Reply:
Resolving.